ludwig/schema/utils.py

```diff
@@ -803,83 +791,6 @@ def _jsonschema_type_mapping(self):
     )
 
 
-@DeveloperAPI
-def InitializerOrDict(
```

Comment:
Jotting it down (as discussed in our sync @justinxzhao ) - this change will remove the ability for users to pass arbitrary dictionaries of values as initialization parameters.